### PR TITLE
Remove the remaining assets

### LINKED ISSUE #7: Remove Assets
Each module will define its assets. We don't need to provide a general asset mechanism in Foundry.

- [x] Remove asset-related RPCs
- [x] Remove asset-related transactions
- [x] Remove the types related to asset from `types`
    - TransferAsset
        - struct AssetTransferInput
        - struct AssetOutPoint
        - struct AssetTransferOutput
    - struct AssetMintOutput    
    - struct AssetWrapCCCOutput
- [x] Remove assets from MemPoolMinFees in `miner` and `rpc types`
- [x] Remove assets from the e2e tests
- [x] Modify the e2e tests

### PR DESCRIPTION
**Pull request by somniumism**
In the last PR #246 , I have not removed all of the concepts associated with the asset. There were documents, test macro, and json scheme texts related to the asset. I removed assets completely.

resolves #7

### CODE REVIEW

File: spec/CodeChain-Address.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-When transferring CCC or assets, the sender must know the recipient's lock script hash and parameters. An address is a converted form of the lock script hash and parameters, and it has some benefits.
+When transferring CCC or addresses, the sender must know the recipient's lock script hash and parameters. An address is a converted form of the lock script hash and parameters, and it has some benefits.
```
Comment (HoOngEe) [inline spec/CodeChain-Address.md]: This part will be removed entirely when account becomes independent from the host. BTW, replacing `assets` with `addresses` is inappropriate. Let's just remove the `or assets` here for a quick fix.

File: spec/Script.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-Codechain saves the unlock condition for each asset, and anyone who can pass this condition is authorized to use the asset. The unlock condition is represented in a byte array, and decoded as a list of instructions before execution. Script language is intentionally designed to not be Turing-complete, and all scripts are ensured to be finished in finite time.
+Codechain saves the unlock condition for each address, and anyone who can pass this condition is authorized to use the address. The unlock condition is represented in a byte array, and decoded as a list of instructions before execution. Script language is intentionally designed to not be Turing-complete, and all scripts are ensured to be finished in finite time.
```
Comment (HoOngEe) [inline spec/Script.md]: Ditto. `address` is not an appropriate substitute for `asset`

File: spec/CodeChain-Address.md (hunk 1 above)
Comment (somniumism) [reply spec/CodeChain-Address.md]: I modified it. : )

File: spec/Script.md (hunk 2 above)
Comment (somniumism) [reply spec/Script.md]: I modified it. : D

**Pull request merged**